From 14f4eb6fb4998cc781cf4c1389d951eb0678ca45 Mon Sep 17 00:00:00 2001
From: Winson Chung <winsonc@google.com>
Date: Mon, 4 Oct 2021 21:50:31 +0000
Subject: [PATCH] DO NOT MERGE: Revert "Map TYPE_TRUSTED_APPLICATION_OVERLAY to
 system window type for A11y"

This reverts commit 4ec7b5c3fbd80d38f1010631dfa84c1476f7c867.

Bug: 196389741
Test: Ensure PIP windows are trusted overlays

Change-Id: I49477ffd489ecbc969ab9b9310c2344ab03280e8
(cherry picked from commit 0cf0d420e18ebb88668f01595ef6acfe3b568430)
Merged-In: I49477ffd489ecbc969ab9b9310c2344ab03280e8
---
 .../server/accessibility/AccessibilityWindowManager.java       | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/services/accessibility/java/com/android/server/accessibility/AccessibilityWindowManager.java b/services/accessibility/java/com/android/server/accessibility/AccessibilityWindowManager.java
index d15c60b9501d..468e93a8f683 100644
--- a/services/accessibility/java/com/android/server/accessibility/AccessibilityWindowManager.java
+++ b/services/accessibility/java/com/android/server/accessibility/AccessibilityWindowManager.java
@@ -725,8 +725,7 @@ public class AccessibilityWindowManager {
                 case WindowManager.LayoutParams.TYPE_SYSTEM_ERROR:
                 case WindowManager.LayoutParams.TYPE_SYSTEM_OVERLAY:
                 case WindowManager.LayoutParams.TYPE_APPLICATION_OVERLAY:
-                case WindowManager.LayoutParams.TYPE_SCREENSHOT:
-                case WindowManager.LayoutParams.TYPE_TRUSTED_APPLICATION_OVERLAY: {
+                case WindowManager.LayoutParams.TYPE_SCREENSHOT: {
                     return AccessibilityWindowInfo.TYPE_SYSTEM;
                 }
 
-- 
2.40.0.rc1.284.g88254d51c5-goog

